frontend/templates/cucumber/.github/workflows/browser.yml

```
@@ -0,0 +1,42 @@
+name: Run Browser Tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
+
+jobs:
+  browser:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+
+    steps:
+      - name: Step 1 - Checkout main branch from GitHub
```

```suggestion
      - name: Checkout main branch from GitHub
```
This won't be the first step since there will be setup steps before, plus other steps are not numbered so this can be consistent